### PR TITLE
Add a section about the Cloud Scalability Model

### PR DESCRIPTION
**Pull request by itaiad200**

### CODE REVIEW

**Comment by kesarwam** [comment]
This doesn't sound right: "1500 read operations per second for all branches combined" if we provide "3000 read operations per second for each branch"

**Comment by itaiad200** [comment]
> This doesn't sound right: "1500 read operations per second for all branches combined" if we provide "3000 read operations per second for each branch"

@kesarwam Changed the order, let me know if it makes more sense

File: docs/understand/lakefs-cloud.md [hunk 1]
```diff
@@ -56,3 +56,12 @@ Please follow the self-service setup wizard on [lakeFS Cloud](https://lakefs.clo
 
 ### GCP
 Please [contact us](mailto:support@treeverse.io) for onboarding instructions.
+
+## Scalability Model
+
+In lakeFS Cloud, a branch is the performance isolation unit.
+The defaults for a lakeFS Cloud installation is 1500 write operations per second and
+1500 read operations per second for all branches combined.
+This limit could be increased by contacting [support]((mailto:support@treeverse.io)).
```
Comment (N-o-Z) [inline docs/understand/lakefs-cloud.md:65]:
```suggestion
This limit can be increased by contacting [support]((mailto:support@treeverse.io)).
```

**Comment by N-o-Z** [comment]
> > This doesn't sound right: "1500 read operations per second for all branches combined" if we provide "3000 read operations per second for each branch"
> 
> @kesarwam Changed the order, let me know if it makes more sense

Still very confusing

**Review by N-o-Z — CHANGES_REQUESTED** [review]
The 1500-3000 thing is very confusing.
Added inline comments

**Review by N-o-Z — COMMENTED** [review]
Still doesn't make sense to me,
If we have a total of 1500 reads per second for all the branches combined how can we get 3000 reads per second per branch?

**Comment by N-o-Z** [comment]
> > Still doesn't make sense to me, If we have a total of 1500 reads per second for all the branches combined how can we get 3000 reads per second per branch?
> 
> The configuration starts from 1500 reads per second for all the branches combined. It could be increased to 40,000 if needed

I think this should be phrased like that in the configuration, I'm guessing that if I didn't understand it, so do the users

File: docs/understand/lakefs-cloud.md [hunk 2]
```diff
@@ -56,3 +56,12 @@ Please follow the self-service setup wizard on [lakeFS Cloud](https://lakefs.clo
 
 ### GCP
 Please [contact us](mailto:support@treeverse.io) for onboarding instructions.
+
+## Scalability Model
```
Comment (ozkatz) [inline docs/understand/lakefs-cloud.md:60]: Data is all here! I would phrase it a little differently though:

```
By default, a lakeFS Cloud installation supports:
- 1,500 read operations/second across all branches on all repositories within a region
- 1,500 write operations per second across all branches on all repositories within a region

This limit can be increased by contacting [support]((mailto:support@treeverse.io)). 

Each lakeFS branch can sustain up to a maximum of 1,000 write operations/second and 3,000 read operations per second. This scales horizontally, so for example, with 10 concurrent branches, a repository could sustain 10k writes/second and 30k reads/second, assuming load is distributed evenly between them.

Reading committed data (e.g. from a commit ID or tag) could be scaled up horizontally to any desired capacity, and defaults to ~5,000 reads/second.

```

Comment (ozkatz) [reply docs/understand/lakefs-cloud.md:60]: @itaiad200 wdyt?

**Review by ozkatz — COMMENTED** [review]
Data is all there, however I think the phrasing could be improved. See my comment for suggestions

File: docs/understand/lakefs-cloud.md (hunk 2 above)
Comment (itaiad200) [reply docs/understand/lakefs-cloud.md:60]: Took it as is, thx!

**Review by N-o-Z — APPROVED** [review]
LGTM!

**Comment by itaiad200** [comment]
Docs check is broken, merging..

**Pull request merged**